server/config/config.go

```diff
@@ -238,6 +238,13 @@ const (
 	defaultGCTunerThreshold           = 0.6
 	minGCTunerThreshold               = 0
 	maxGCTunerThreshold               = 0.9
+
+	defaultEnableKeyspaceGC   = false
+	defaultKeyspaceGCInterval = 10 * time.Minute
+	minKeyspaceGCInterval     = 10 * time.Minute
+	maxKeyspaceGCInterval     = 24 * time.Hour
+	defaultKeyspaceGCLifeTime = 7 * 24 * time.Hour
+	minKeyspaceGCLifeTime     = 1 * time.Hour
```

Comment:
An hour is a bit long. In a test environment, we usually want to clean up as early as possible.